tests/exposure_analysis_tests/test_allow_all/netpol.yaml

```diff
@@ -0,0 +1,16 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-conns-to-all-world
+  namespace: hello-world
+spec:
+  podSelector:
+    matchLabels:
+      app: a-app
+  policyTypes:
+  - Ingress
+  - Egress
+  ingress:
+  - {}
+  egress:
+  - {}
```

Comment:
could be more interesting if one side blocks and the other not , such as:
```
  policyTypes:
  - Ingress
  - Egress
  egress:
  - {}
```

Reply:
- added, all these tests and more tests: will be added for output testing too.
- fixes to exposureMap (from my local "textual output branch" were committed to here); fixing adding the correct `protected data` of a relevant peer  /its new entry in the map

Reply:
hi @adisos , committed a new fix of splitting the exposure map into two ingress and egress exposure maps